### PR TITLE
File modified and about to commit

### PR DESCRIPTION
**Pull request by ninaddaudkhane**
Add checklist

### CODE REVIEW

File: ninaddaudkhane-minneapolis.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+First change in this file!!
+
+Places to eat:
+
+1) Spoon and Stable
```
Comment (githubteacher) [inline ninaddaudkhane-minneapolis.md]: To create a numbered list, use the following format:

1. Spoon and Stable
1. Houlihaans

etc...

File: ninaddaudkhane-minneapolis.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+2) Houlihaans
+3) Natures Basket
```
Comment (githubteacher) [inline ninaddaudkhane-minneapolis.md]: What is your favorite thing to eat at Natures Basket? I really like 🌮 s and 🌯 s, do they have good ones?

**Review by uppuravi — CHANGES_REQUESTED** [review]
git

File: ninaddaudkhane-minneapolis.md (hunk 2 above)
Comment (githubteacher) [inline ninaddaudkhane-minneapolis.md]: If you want to add places to visit, can you provide some awesome craft breweries, I've already been to Tin Whiskers in St. Paul 😄 🍺

**Review by githubteacher — CHANGES_REQUESTED** [review]
@ninaddaudkhane please review the requested changes and make modifications to your file.

File: ninaddaudkhane-minneapolis.md (hunk 2 above)
Comment (ninaddaudkhane) [reply ninaddaudkhane-minneapolis.md]: You can try Blue Moon !!

File: ninaddaudkhane-minneapolis.md (hunk 1 above)
Comment (uppuravi) [inline ninaddaudkhane-minneapolis.md]: cool

**Review by uppuravi — CHANGES_REQUESTED** [review]
git

**Review by githubteacher — APPROVED** [review]
@ninaddaudkhane this looks great. ready to 🚢  ... but not yet.

**Pull request merged**